src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -2305,6 +2305,7 @@ private void SetPeriodicEvaluation(WorkspaceModel ws)
                 this.LinterManager);
 
             workspace.FileName = string.IsNullOrEmpty(filePath) ? "" : filePath;
+            workspace.FileContentId = string.IsNullOrEmpty(filePath) ? fileContents.GetHashCode().ToString() : "";
```

Comment:
I don't think this will generate stable hashcodes in .net6+.
@aparajit-pratap found that the strings no longer generate the same hashcode in different sessions. Can you find the docs on that @aparajit-pratap ?

Reply:
Yes but the usage of FileContendId is per session only and it is not serialized. But the GetHashCode implemented in Dictionary is nicer and makes things more clear so I'll use that one.